integration-tests/jest.config.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   verbose: false,
   preset: 'jest-puppeteer',
   testEnvironment: '<rootDir>/jest-circus-environment.js',
-  testRegex: '\\.integration-test\\.js$',
+  testMatch: ['<rootDir>/tests/*.js'],
```

Comment:
Glob seemed a little easier to read than regex.

https://jestjs.io/docs/configuration#testmatch-arraystring